css/icons.css

```diff
@@ -92,29 +92,28 @@
  * .app-Talk rules above.
  * "forced-white" needs to be included in the class name as the Avatar does
  * not accept several classes. */
-.user-bubble__avatar .icon-group-forced-white,
-.user-bubble__avatar .icon-user-forced-white,
-.autocomplete-result .icon-group-forced-white,
-.autocomplete-result .icon-user-forced-white,
-.mention-bubble .icon-group-forced-white,
-.mention-bubble .icon-user-forced-white {
-	background-color: var(--color-text-maxcontrast-default) !important;
-}
-
-body[data-theme-dark] .icon-group-forced-white,
-body[data-theme-dark] .icon-user-forced-white {
-	background-color: #3B3B3B !important;
+.user-bubble__avatar .icon-group-forced-white.avatar-class-icon,
+.user-bubble__avatar .icon-user-forced-white.avatar-class-icon,
+.autocomplete-result .icon-group-forced-white.autocomplete-result__icon--,
+.autocomplete-result .icon-user-forced-white.autocomplete-result__icon--,
+.mention-bubble .icon-group-forced-white.mention-bubble__icon--,
+.mention-bubble .icon-user-forced-white.mention-bubble__icon-- {
+	background-color: #6B6B6B;
+	background-size: 75%;
 }
 
-.user-bubble__avatar .icon-group-forced-white,
-.user-bubble__avatar .icon-user-forced-white {
+body[data-theme-dark] .user-bubble__avatar .icon-group-forced-white.avatar-class-icon,
+body[data-theme-dark] .user-bubble__avatar .icon-user-forced-white.avatar-class-icon,
+body[data-theme-dark] .autocomplete-result .icon-group-forced-white.autocomplete-result__icon--,
+body[data-theme-dark] .autocomplete-result .icon-user-forced-white.autocomplete-result__icon--,
+body[data-theme-dark] .mention-bubble .icon-group-forced-white.mention-bubble__icon--,
+body[data-theme-dark] .mention-bubble .icon-user-forced-white.mention-bubble__icon-- {
+	background-color: #3B3B3B;
```

Comment:
right sidebar is 3B with dark mode, but the group mention in the chat is still 6B on dark-mode.

After some time I then finally found it...
`body[data-theme-dark]` does not match.... My theme is empty as I use "system default" instead of hardcoded dark mode